lib/sites-base.js

```diff
@@ -201,7 +201,7 @@ module.exports = {
       if ((!doc._id) && (!doc.copyOfId)) {
         if (!doc.adminPassword) {
           self.apos.notify(req, 'You must fill out the admin password field when creating a new site.', { type: 'error' });
-          return callback(new Error('adminPassword.required'));
+          return callback('adminPassword.required');
```

Comment:
Etienne changed this to "new Error" because that's more common modern practice but that's not how we do it in A2 and there is code that at least tries to do something with the string.